docs/develop/json-manifest-overview.md

```diff
@@ -1,40 +1,41 @@
 ---
-title: Teams manifest for Office Add-ins (preview)
-description: Get an overview of the preview JSON manifest.
+title:  Unified Microsoft 365 manifest (preview)
+description: Get an overview of the preview unified Microsoft 365 manifest.
 ms.topic: overview
 ms.date: 06/15/2022
 ms.localizationpriority: high
 ---
 
-# Teams manifest for Office Add-ins (preview)
+# Unified Microsoft 365 manifest (preview)
 
 Microsoft is making a number of improvements to the Microsoft 365 developer platform. These improvements provide more consistency in the development, deployment, installation, and administration of all types of extensions of Microsoft 365, including Office Add-ins. These changes are compatible with existing add-ins.
 
-One important improvement we are working on is the ability to create a single unit of distribution for all your Microsoft 365 extensions by using the same manifest format and schema, based on the current JSON-formatted Teams manifest.
+One important improvement we are working on is the ability to create a single unit of distribution for all your Microsoft 365 extensions by using the same manifest format and schema, based on the current JSON-formatted Unified Microsoft 365 manifest.
```

Comment:
Should this be "JSON-formatted unified Microsoft 365 manifest" or will the Teams documentation use the capitalized version? Because the Teams dev documentation still refers to the manifest as "Microsoft Teams app manifest", should we say something like "JSON-formatted unified Microsoft 365 manifest (previously known as Microsoft Teams apps manifest)"?

Reply:
```suggestion
One important improvement we are working on is the ability to create a single unit of distribution for all your Microsoft 365 extensions by using the same manifest format and schema, based on the current JSON-formatted unified Microsoft 365 manifest.
```